### PR TITLE
Windows error codes as non-exhaustive enums

### PR DESCRIPTION
**Pull request by daurnimator**

### CODE REVIEW

File: lib/std/os/windows/bits.zig [hunk 1]
```diff
@@ -929,9 +928,12 @@ pub usingnamespace switch (builtin.arch) {
             SegSs: DWORD,
             ExtendedRegisters: [512]BYTE,
 
-            pub fn getRegs(ctx: *const CONTEXT) struct {bp: usize, ip: usize} {
-                return .{.bp = ctx.Ebp, .ip = ctx.Eip};
-            }  
+            pub fn getRegs(ctx: *const CONTEXT) struct {
+                bp: usize,
+                ip: usize,
```
Comment (daurnimator) [inline lib/std/os/windows/bits.zig]: This change was made by `zig fmt` and it slipped into this PR.
@LemonBoy did #4342 fix this unwanted change?

Comment (LemonBoy) [reply lib/std/os/windows/bits.zig]: This is another different and unrelated problem `zig fmt` has.

Comment (daurnimator) [reply lib/std/os/windows/bits.zig]: Should I make an issue for it?

Comment (daurnimator) [reply lib/std/os/windows/bits.zig]: https://github.com/ziglang/zig/issues/4349

**Pull request merged**